contracts/modules/cover/CoverMigrator.sol

```diff
@@ -20,16 +20,33 @@ contract CoverMigrator is MasterAwareV2 {
     return ICover(getInternalContractAddress(ID.CO));
   }
 
+  function individualClaims() internal view returns (IIndividualClaims) {
+    return IIndividualClaims(getInternalContractAddress(ID.IC));
+  }
+
+  function coverNFT() internal returns (ICoverNFT) {
+    return ICoverNFT(cover().coverNFT());
+  }
+
   /// @dev Migrates covers for arNFT-like contracts that don't use Gateway.sol
   ///
   /// @param coverId          Legacy (V1) cover identifier
-  function submitClaim(uint coverId) external whenNotPaused {
-    cover().migrateCoverFromOwner(coverId, msg.sender, tx.origin);
+  function submitClaim(
```

Comment:
I don't expect this to work anymore given it's original purpose.

https://github.com/NexusMutual/smart-contracts/blob/master/contracts/modules/claims/Claims.sol#L82

CoverMigrator was replacing Claims and whatever contracts would call submitClaim on Claims.sol would now hit this function.

The signature is now changed so it won't work.

We can have 2 separate functions here if this new one still helps with UX in some cases.